tests/foreman/longrun/test_n_1_upgrade.py

```diff
@@ -0,0 +1,262 @@
+"""Test class for N-1 upgrade feature
+
+:Requirement: Satellite one version ahead from Capsule
+
+:CaseAutomation: ManualOnly
+
+:CaseLevel: Integration
```

Comment:
Since all your tests have `:CaseLevel: System`, you could set this field to `System` here and drop level from individual tests.